bazel_example/BUILD.bazel

```diff
@@ -9,7 +10,7 @@ proto_library(
 
 csharp_proto_library(
     name = "example_proto",
```

Comment:
Please rename this to `"example_csharp_proto"` according to bazel naming conventions

Reply:
done